aten/src/ATen/native_parse.py

```diff
@@ -78,6 +78,10 @@ def type_argument_translations(arg):
         t = 'double'
     elif t == 'float?':
         t = 'double?'
+    elif t == 'float[]':
```

Comment:
oh, I think this is where you needed the "using" from the IntArrayRef? PR.  Better if you keep them together, but not a big deal.